pyproject.toml

```diff
@@ -152,53 +152,31 @@ extend-exclude = """(
 # Same as Black. + 10% rule of black
 line-length = 88
 
-# Enable pycodestyle (`E`) and Pyflakes (`F`) codes by default.
-# (`I`) is for `isort`.
-select = ["E", "F", "I"]
+# Exclude generated files.
+extend-exclude = [
+    "lnbits/wallets/lnd_grpc_files"
+]
+
+[tool.ruff.lint]
+# Enable:
+# F - pyflakes
+# E - pycodestyle errors
+# W - pycodestyle warnings
+# I - isort
+select = ["F", "E", "W", "I"]
 ignore = []
 
 # Allow autofix for all enabled rules (when `--fix`) is provided.
 fixable = ["ALL"]
 unfixable = []
 
-# Exclude a variety of commonly ignored directories.
```

Comment:
How come these directories do not have to be excluded anymore?

Reply:
because i changed `exclude` to `extend-exclude` which keeps the default excluded dirs only extends it

Reply:
nice!